sw/device/rom_exts/manifest.md

```diff
@@ -30,9 +30,13 @@ Based on the requirements outlined in [boot.md](./boot)
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                       Signature Exponent                      |
```

Comment:
Just curious, why is the exponent not next to modulus, i.e. `Signature Public Key (3072 bits)`?

Reply:
Alignment requirements, trying to keep 32-bit stuff together.